packages/common-ts/src/base-service/validators.ts

```diff
@@ -18,6 +18,13 @@ const provider = makeValidator<Provider>((input) => {
   return new ethers.providers.JsonRpcProvider(parsed)
 })
 
+const jsonRpcProvider = makeValidator<ethers.providers.JsonRpcProvider>(
+  (input) => {
+    const parsed = url()._parse(input)
+    return new ethers.providers.JsonRpcProvider(parsed)
```

Comment:
What do you think about making this a `StaticJsonRpcProvider` to save some http requests for `eth_chainId`?

Reply:
Added both.